process/smartContract/builtInFunctions/factory.go

```diff
@@ -21,8 +21,8 @@ type ArgsCreateBuiltInFunctionContainer struct {
 	ShardCoordinator     sharding.Coordinator
 }
 
-// NewBuiltInFunctionsFactory creates a factory which will instantiate the built in functions contracts
-func NewBuiltInFunctionsFactory(args ArgsCreateBuiltInFunctionContainer) (vmcommon.BuiltInFunctionContainer, error) {
+// CreateBuiltInFunctionContainer creates a factory which will instantiate the built in functions contracts
```

Comment:
CreateBuiltInFunctionContainer creates a container that will hold all the available built in functions